migrations/20230613116515_courses.sql

```diff
@@ -0,0 +1,103 @@
+-- Courses Table
+CREATE TABLE IF NOT EXISTS public.courses (
+    id SERIAL PRIMARY KEY,
+    catalog_number SMALLINT NOT NULL,
+    subject_code VARCHAR(10) NOT NULL,
+    external_id VARCHAR(10) NOT NULL,
+    academic_level VARCHAR(13) NOT NULL,
```

Comment:
I think it would be better if this were an Enum.

Reply:
Enums in Rust should probably be Enums in the DDL 

https://www.postgresql.org/docs/current/datatype-enum.html